packages/cpt-ui/src/pages/PrivacyNoticePage.tsx

```diff
@@ -0,0 +1,187 @@
+            <li>{PrivacyNoticeStrings.usage.legal}</li>
+          </ul>
+
+          <h2 className="nhsuk-heading-l">
+            {PrivacyNoticeStrings.legal.header}
+          </h2>
+          <p>{PrivacyNoticeStrings.legal.basis}</p>
+          <ol>
+            <li>
+              {PrivacyNoticeStrings.legal.base1}
+              <Link to={contactInfo.links.legalLink}>
+                {PrivacyNoticeStrings.legal.basesLink}</Link>
+            </li>
+            <li>
+              {PrivacyNoticeStrings.legal.base2}
+              <Link to={contactInfo.links.legalLink}>
+                {PrivacyNoticeStrings.legal.basesLink}</Link>
+            </li>
+          </ol>
+          <p>
+            {PrivacyNoticeStrings.legal.note}
+            <Link to={contactInfo.links.legalLink}>
+              {PrivacyNoticeStrings.legal.basesLink}</Link>
+          </p>
+
+          <h2 className="nhsuk-heading-l">{PrivacyNoticeStrings.sharing.header}</h2>
+          <p>
+            {PrivacyNoticeStrings.sharing.mayShare}
+          </p>
+          <ul>
+            <li>
+              {PrivacyNoticeStrings.sharing.share1}
+            </li>
+            <li>
+              {PrivacyNoticeStrings.sharing.share2}
+            </li>
+          </ul>
+          <h2 className="nhsuk-heading-l">{PrivacyNoticeStrings.retention.header}</h2>
+          <p>
+            {PrivacyNoticeStrings.retention.note}
+          </p>
+          <ul>
+            <li>
+              {PrivacyNoticeStrings.retention.account}
+            </li>
+            <li>
+              {PrivacyNoticeStrings.retention.system}
+            </li>
+            <li>
+              {PrivacyNoticeStrings.retention.analytics}
+            </li>
+          </ul>
+
+          <h2 className="nhsuk-heading-l">
+            {PrivacyNoticeStrings.storage.header}
+          </h2>
+          <p>
+            {PrivacyNoticeStrings.storage.body}
+          </p>
+
+          <h2 className="nhsuk-heading-l">
+            {PrivacyNoticeStrings.rights.header}</h2>
+          <p>
+            {PrivacyNoticeStrings.rights.contact}
+            {createEmailLink(contactInfo.emails.dpo)}
+          </p>
+          <ol>
+            <li>
+              {PrivacyNoticeStrings.rights.rights1}
+            </li>
+            <li>
+              {PrivacyNoticeStrings.rights.rights2}
+              <Link to={contactInfo.links.personalData}>
+                {PrivacyNoticeStrings.rights.rights2Link}
+              </Link>
+            </li>
+            <li>
+              {PrivacyNoticeStrings.rights.rights3}
+            </li>
+            <li>
+              {PrivacyNoticeStrings.rights.rights4}
+            </li>
+            <li>
+              {PrivacyNoticeStrings.rights.rights5}
+              <Link to="/cookies">
+                {PrivacyNoticeStrings.rights.cookieLink}
+              </Link>
+            </li>
+          </ol>
+
+          <div className="nhsuk-inset-text">
```

Comment:
https://main--65aa76b29d00a047fe683b95.chromatic.com/?path=/docs/content-presentation-insettext--docs

Should use the component